Code/Framework/AzCore/AzCore/IO/Streamer/RequestPath.cpp

```diff
@@ -142,14 +129,15 @@ namespace AZ
             }
         }
 
-        size_t RequestPath::FindAliasOffset(const AZStd::string& path) const
+        size_t RequestPath::FindAliasOffset(AZStd::string_view path) const
         {
-            const char* pathChar = path.c_str();
+            const char* pathChar = path.data();
+            const char* pathCharEnd = path.end();
```

Comment:
This indicates Filesystem Alias Resolver class I am thinking of might need to have a "SplitAliasAndRelativePath" function that returns a tuple of path views to the alias segment of a path and post alias segment.
Thoughts?

Reply:
Instead of an `AZStd::tuple` an `AZStd::pair` or a dedicated struct, but otherwise yes, that would be incredibly useful.